resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -595,7 +595,7 @@ private[spark] class ApplicationMaster(
         }
         failureCount = 0
       } catch {
-        case i: InterruptedException => // do nothing
+        case _: InterruptedException => // do nothing
```

Comment:
These are not important, but OK if we're just doing cleanup. Seems OK

Reply:
Yeah, if it's not important, I can roll back these. They are in a commit

Reply:
@srowen [53a6bfa](https://github.com/apache/spark/pull/39997/commits/53a6bfa6373872e3a8edaa531486d82e593d0ea7) revert [92f9b38](https://github.com/apache/spark/pull/39997/commits/92f9b38d0641d3db7c751e9d155fbacb3df776a2) and update pr description